src/Schema/Schema.php

```diff
@@ -757,12 +811,17 @@ protected function getCacheTag(): string
      * @param string $type metadata type.
      * @param bool $refresh whether to reload the table metadata even if it is found in the cache.
      *
+     * @throws JsonException
+     *
      * @return mixed metadata.
      */
     protected function getTableMetadata(string $name, string $type, bool $refresh = false)
     {
-        if ($this->db->isSchemaCacheEnabled() && !\in_array($name, $this->db->getSchemaCacheExclude(), true)) {
-            $schemaCache = $this->cache;
+        if (
+            $this->schemaCache->isCacheEnabled() &&
+            !in_array($name, $this->schemaCache->getCacheExclude(), true)
```

Comment:
This check could be moved into SchemaCache::isExcluded($name). Then we can remove `getCacheExclude()`.

Reply:
Done.